templates/zerver/help/configure-audible-notifications.md

```diff
@@ -1,18 +1,18 @@
 # Configure audible notifications
 
 You can configure your settings to receive audible notifications for messages
-sent when Zulip is offscreen.
+sent while Zulip is offscreen.
 
 {!go-to-the.md!} [Notifications](/#settings/notifications)
 {!settings.md!}
 
-2.    If you want to receive notifications for each new message from a stream,
-select the **Audible notifications** option under **Stream messages**.
+    * If you want to have audible notifications for each new message from a
```

Comment:
No, I just thought it looks better that way. I guess I'll put it without indent (that might be worth getting into the styleguide docs)